### PR TITLE
Feature/86cuwqpa7 booking details page

### PR DESCRIPTION
**Pull request by NikkiShao**
Completed booking details page and related database connection for artist and bride profiles
- artist and bride profile now loads bookings from database (need to load the new mock data - bookings12.json to test)
- buttons on page to interact change based on status and user type

image
image

note: Some locations of the mock data is fake and thus cannot be found with mapbox (will display coordinates 0,0 in the ocean)

### CODE REVIEW

File: behind-the-veil-siteroot/imports/api/collections/booking.js
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/api/collections/booking.js:1]: probably should rename the file to bookings

**Review by nethbtju — DISMISSED** [review]
Need to change file names but otherwise approved

File: behind-the-veil-siteroot/imports/api/collections/booking.js
Comment (NikkiShao) [reply behind-the-veil-siteroot/imports/api/collections/booking.js:1]: oki

**Review by nethbtju — APPROVED** [review]
No issues all approved

**Review by nethbtju — APPROVED** [review]

**Pull request merged**